### PR TITLE
add Travis

### PR DESCRIPTION
**Pull request by exussum12**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: php
+dist: trusty
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+    - $HOME/.cache/composer/files
+matrix:
+  include:
+    - php: 5.6
```
Comment (michaelcullum) [inline .travis.yml]:
```suggestion
    - php: 5.6
    - php: 7.3
  allow_failures:
    - php: 7.3
```

Comment (svpernova09) [reply .travis.yml]: IMHO we should not let 7.3 fail currently as we need to know it's status since we should be targeting 7.3 with the new server/setup. We should add & allow `nightly` to fail.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  fast_finish: true
+before_script:
+  - composer global require squizlabs/php_codesniffer 
+  - composer global require phpunit/phpunit
```
Comment (michaelcullum) [inline .travis.yml]: These should really be composer dev dependencies (at least phpunit) and we should adjust build.sh to use `./vendor/bin/phpunit` but happy to leave that for a separate PR

File: .travis.yml (hunk 1 above)
Comment (michaelcullum) [reply .travis.yml]: 7.3 will fail until compatibility is finished though and we shouldn't block merging all PRs until then. Setting off on a note of it being okay to merge failing PRs is non-ideal I think.

We would then remove `allow_failures: 7.3` in the compatibility PR (and remove 5.6).

Comment (svpernova09) [reply .travis.yml]: Sounds good!

File: .travis.yml (hunk 2 above)
Comment (iansltx) [reply .travis.yml]: +1 for separate PR, after #750 lands
